test/ssl_test_ctx.c

```diff
@@ -286,6 +286,29 @@ const char *ssl_session_ticket_name(ssl_session_ticket_t server)
                      server);
 }
 
+/* CompressionExpected */
+
+static const test_enum ssl_compression[] = {
```

Comment:
Good question. Apparently there already is a built-in boolean type that I should have been using.